docs/root/version_history/current.rst

```diff
@@ -143,6 +143,7 @@ New Features
 * http: added new runtime config `envoy.reloadable_features.check_unsupported_typed_per_filter_config`, the default value is true. When the value is true, envoy will reject virtual host-specific typed per filter config when the filter doesn't support it.
 * http: added the ability to preserve HTTP/1 header case across the proxy. See the :ref:`header casing <config_http_conn_man_header_casing>` documentation for more information.
 * http: change frame flood and abuse checks to the upstream HTTP/2 codec to ON by default. It can be disabled by setting the `envoy.reloadable_features.upstream_http2_flood_checks` runtime key to false.
+* http: hash multiple header values instead of only hash the first header value. It can be disabled by setting the `envoy.reloadable_features.hash_multiple_header_values` runtime key to false.
```

Comment:
I'm not very sure whether to put it, I added as a new feature. But also feel like minor behavior change. Let me know the correct place, we will update it.

Reply:
I would put this in minor behavior changes. Also, please ref link to the configuration field where this is configured.

Reply:
thanks, updated